addon/lib/shooter.js

```diff
@@ -377,11 +392,12 @@ const ShotContext = Class({
       } catch (e) {
         // Ignore... it's just a best effort to cancel the state
       }
-      this.interactiveWorker.destroy();
+      setTimeout(() => {
+        this.interactiveWorker.destroy();
+        this.interactiveWorker = null;
+      }, 1000);
```

Comment:
What's going on here?  (I think I might have gotten an exception related to this in testing, but I'm not sure)